packages/styles/src/components/form-select.scss

```diff
@@ -9,14 +9,43 @@
 @use './../variables/components/forms';
 @use './../variables/components/form-select' as form-select;
 
+.form-select-wrapper {
+  @include utilities.focus-style;
+}
+
+:not(.form-select-wrapper) > {
+  .form-select {
+    @include utilities.focus-style;
+  }
+}
+
```

Comment:
We can either do it like that or add a breaking change so that user needs to add a wrapper element